src/spatial/collision.js

```diff
@@ -353,7 +346,7 @@ Crafty.c("Collision", {
     /**@
```

Comment:
I think It was included on purpose, but if It was made by mistake I agree that removing this method from public API is good practise. That solve?